core/src/main/java/io/github/tcdl/msb/monitor/aggregator/DefaultChannelMonitorAggregator.java

```diff
@@ -71,31 +74,52 @@ void onHeartbeatResponses(List<Message> heartbeatResponses) {
         for (Message msg : heartbeatResponses) {
             aggregateInfo(aggregatorStats, msg);
         }
-        LOG.debug(String.format("Calling registered handler for heartbeat %s...", heartbeatResponses));
+        LOG.debug(String.format("Calling registered handler for aggregated statistics %s...", masterAggregatorStats));
```

Comment:
I don't see the reason to have identical log messages for announcements and heartbeats. Different messages might ease problem investigation and don't force you to copy/paste them.

If you really want to make the identical I would suggest to extract LOG.debug and handler.call into a separate method and call it twice.